v3-sdk/trading/.eslintrc.json

```diff
@@ -16,9 +16,12 @@
         "sourceType": "module"
     },
     "plugins": [
-        "typescript-sort-keys"
+        "typescript-sort-keys",
+        "simple-import-sort"
     ],
     "rules": {
+        "simple-import-sort/imports": "error",
+        "simple-import-sort/exports": "error",
```

Comment:
Can we add these all at once? I think it makes sense to do another PR for all examples for this if we do it at all but don't think it's critical to include

Reply:
This example had a bunch of imports and this was really needed. Let's keep it here and I can update the rest too?

Reply:
Okay, let's fast follow PR then please! I just didn't want to require more work to ship this and leaving this off as its own task would make this quicker. I don't think it adds value really still for a user, really only for maintainability.